script/RumpelConfig.sol

```diff
@@ -630,6 +657,248 @@ library RumpelConfig {
+        ProtocolGuardConfig[] memory configs = new ProtocolGuardConfig[](17);
+
+        // Mellow Simple Lido DV
+        configs[0] = ProtocolGuardConfig({target: MAINNET_MELLOW_DVSTETH, selectorStates: new SelectorState[](2)});
+        configs[0].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[0].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow pzETH
+        configs[1] = ProtocolGuardConfig({target: MAINNET_MELLOW_RENZO_PZETH, selectorStates: new SelectorState[](2)});
+        configs[1].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[1].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow rsENA
+        configs[2] = ProtocolGuardConfig({target: MAINNET_MELLOW_RSENA, selectorStates: new SelectorState[](2)});
+        configs[2].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[2].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow rsENA
+        configs[3] = ProtocolGuardConfig({target: MAINNET_MELLOW_AMPHRBTC, selectorStates: new SelectorState[](2)});
+        configs[3].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[3].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow steakLRT
+        configs[4] = ProtocolGuardConfig({target: MAINNET_MELLOW_STEAKLRT, selectorStates: new SelectorState[](2)});
+        configs[4].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[4].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+
+        // Mellow HYVEX
+        configs[5] = ProtocolGuardConfig({target: MAINNET_MELLOW_HYVEX, selectorStates: new SelectorState[](2)});
+        configs[5].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
+        configs[5].selectorStates[1] =
+            SelectorState({selector: IMellow.registerWithdrawal.selector, state: RumpelGuard.AllowListState.ON});
+        
+        // Mellow Re7rtBTC
+        configs[6] = ProtocolGuardConfig({target: MAINNET_MELLOW_RE7RTBTC, selectorStates: new SelectorState[](2)});
```

Comment:
I believe the deposit asset here is `tBTC`, so if we want to add this vault we should add transfers for that token, too (most of these strats use wstETH, which is already added, but there might be others that use deposit tokens that we haven't yet – worth doing a quick double check)

Reply:
also! just remembered that I believe @stevenvaleri just wrote a script to fetch and nicely format the current state of the allow/block lists https://github.com/sense-finance/rumpel-wallet/tree/feat/print-current-state